packages/native/types/tslint.json

```diff
@@ -1,20 +1,24 @@
 {
-    "extends": "dtslint/dtslint.json",
-    "rules": {
-        "array-type": [true, "generic"],
-        "import-spacing": false,
-        "semicolon": false,
-        "whitespace": [
-            true,
-            "check-branch",
-            "check-decl",
-            "check-operator",
-            "check-module",
-            "check-rest-spread",
-            "check-type",
-            "check-typecast",
-            "check-type-operator",
-            "check-preblock"
-        ]
-    }
+  "extends": "dtslint/dtslint.json",
+  "rules": {
+    "array-type": [true, "generic"],
+    "callable-types": false,
+    "file-name-casing": false,
+    "import-spacing": false,
+    "semicolon": false,
+    "whitespace": [
+      true,
+      "check-branch",
+      "check-decl",
+      "check-operator",
+      "check-module",
+      "check-rest-spread",
+      "check-type",
+      "check-typecast",
+      "check-type-operator",
+      "check-preblock"
+    ],
+    "no-null-undefined-union": false,
+    "strict-export-declare-modifiers": false
+  }
```

Comment:
these rules were copied from `core` and then pared down a bit for rules that didn't need to be disabled. we grew `"callable-types": false` to deal with the `FunctionInterpolation` definition, and the two rules at the bottom to deal with `type Interpolation` and `type ReactNative = typeof RN` respectively.